hooks/relationship_render.py

```
@@ -67,7 +68,275 @@ def _sticky(cx, y, impl, index, name_font, pkg_font, role, wraps):
+    '<feDropShadow dx="2" dy="4" stdDeviation="3.5" flood-color="#000" flood-opacity="0.26"/></filter>'
+    '<marker id="tri" markerWidth="22" markerHeight="16" refX="15" refY="7.5" orient="auto">'
+    '<path d="M2,2 Q10,0.5 15.5,7.5 Q9.5,14 2.5,12.5 Q4,7.5 2,2 Z" fill="#fbfbf5" stroke="#2b2b2b" '
+    'stroke-width="1.5" stroke-linejoin="round" stroke-linecap="round"/></marker>'
+    '<marker id="use" markerWidth="13" markerHeight="13" refX="9" refY="5.5" orient="auto">'
+    '<path d="M1.5,1.5 L10,5.5 L1.5,9.5" fill="none" stroke="#2b2b2b" stroke-width="1.7" '
+    'stroke-linecap="round" stroke-linejoin="round"/></marker>'
+    '</defs>'
+)
+
+
+def _interface_box(cx, top, role, page, rot, font, is_list):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_interface_box`

(ANN202)